web-ts/src/components/LocationHeader.tsx

```diff
@@ -138,10 +142,9 @@ export class LocationHeader extends React.Component<PropsType> {
       items.push(
         <HeaderNavigationItem
           key='offers'
-          // TODO Use right path and check
-          href='/'
           active={false}
-          // href={new OffersRouteConfig().getRoutePath({ city, language })}
+          href={offersPath}
+          // TODO Use right check
```

Comment:
Shall we add a task for this to keep this in mind and link it here and at the other "TODO user right checks"?

Reply:
Created an issue: https://issues.tuerantuer.org/browse/IGAPP-668